Marlin/language_da.h

```diff
@@ -101,6 +102,10 @@
 #define MSG_VZ_JERK                         "Vz-jerk"
 #define MSG_VE_JERK                         "Ve-jerk"
 #define MSG_VMAX                            "Vmax "
+#define MSG_X                               "X"
+#define MSG_Y                               "Y"
+#define MSG_Z                               "Z"
+#define MSG_E                               "E"
```

Comment:
You should remove `MSG_X`, `MSG_Y`, `MSG_Z`, and `MSG_E` (which were intentionally removed).